mon-pix/translations/en.json

```
@@ -874,7 +874,7 @@
           "close": "Ok, got it",
           "focused": {
             "title": "Focus Mode",
-            "content": "'<p class=\"tooltip-tag-information__title--focused\">'Stay on this page to answer!'</p>' Do not use the internet or any application to answer. Any activity outside this zone will be detected."
+            "content": "'<strong class=\"tooltip-tag-information__title--focused\">'Stay on this page to answer!'</strong>' Do not use the internet or any application to answer. Any activity outside this zone will be detected."
```

```suggestion
            "content": "'<strong>'Stay on this page to answer!'</strong><br />' Do not use the internet or any application to answer. Any activity outside this zone will be detected."
```

Il est possible de se passer complètement du CSS spécifique.
Le contenu est déjà en gras grâce à la balise `strong` et on peut revenir à la ligne avec un `br`.